react/src/lib/components/DeckGLMap/storybook/DeckGLMap.stories.jsx

```diff
@@ -302,6 +302,39 @@ KhMapMesh.args = {
     },
 };
 
+//Material property may take these values:
+//          true  = default material,
+//          false = no material. This is default property value.
+//          Full spec:
+//                {
+//                    ambient: 0.35,
+//                    diffuse: 0.6,
+//                    shininess: 32,
+//                    specularColor: [255, 255, 255],
+//                }
+const material = {
+    ambient: 0.35,
+    diffuse: 0.6,
+    shininess: 32,
+    specularColor: [255, 255, 255],
+};
+export const MapMaterial = MinimalTemplate.bind({});
```

Comment:
The story looks narrow on the Docs tab. Maybe look at the Kh Map Flat example to make the example bigger, and add a short paragraph that explains what the example does.

![image](https://user-images.githubusercontent.com/19621822/158345983-92950cb7-7aab-4267-ab35-77844cfb1064.png)